engine/flink/model-util/src/main/scala/pl/touk/nussknacker/engine/flink/util/transformer/aggregate/aggregates.scala

```diff
@@ -78,7 +78,8 @@ object aggregates {
 
     override def mergeAggregates(agg1: Aggregate, agg2: Aggregate): Aggregate = agg1 ++ agg2
 
-    override def result(finalAggregate: Aggregate): AnyRef = new java.util.ArrayList[Any](finalAggregate.asJava)
+    //we revert elements only during final results computation
```

Comment:
Woops. Will this change break a business logic in processes that was made on assumption that the order is different?

Reply:
Good point, I will defer it to next PR. For now, I added TODO